data/core/view.lua

```diff
@@ -46,51 +47,77 @@ function View:get_name()
   return "---"
 end
 
-
+-- Returns `y, x` for compatibility
 function View:get_scrollable_size()
```

Comment:
I think we are abusing this function here and we may break compatibility. Shouldn't we use another function for horizontal scrolling like get_h_scrollable_size ?